src/lib/tabs/tab-nav-bar/tab-nav-bar.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, Input, ViewChild, ElementRef } from '@angular/core';
+import { MdInkBar } from '../ink-bar';
+
+@Component({
+  moduleId: module.id,
+  selector: 'md-tab-nav-bar',
+  templateUrl: 'tab-nav-bar.html',
+  styleUrls: ['tab-nav-bar.css'],
+})
+export class MdTabNavBar {
+  @ViewChild(MdInkBar) private _inkBar: MdInkBar;
+
+  updateActiveLink(element: HTMLElement) {
+    this._inkBar.alignToElement(element);
+  }
+}
+
+@Component({
+  moduleId: module.id,
+  selector: '[md-tab-link]',
+  template: '<ng-content></ng-content>',
+  styleUrls: ['tab-link.css']
+})
+export class MdTabLink {
+  private _isActive: boolean = false;
+
+  @Input()
+  get active(): boolean {
```

Comment:
I didn't know about this before, but looking through the router docs turned up this:
https://angular.io/docs/ts/latest/guide/router.html#!#router-link-active

Seems like we could rely on that to apply the appropriate class based on the route being active